recipes/allennlp-models/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: allennlp-models
+  version: {{ version }}
+
+source:
+  url: https://github.com/allenai/allennlp-models/archive/v{{ version }}.tar.gz
+  sha256: 34c3beb30d26e49e0bd90b2b4913d5af0db785b4e654a3cbbc44efd334dc33e9
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

How about noarch? It's fine for the staged-recipes CI to fail with missing dependencies on other platforms
```suggestion
  noarch: python
```